tests/test_extension.py

```diff
@@ -0,0 +1,92 @@
+try:
+    import unittest.mock as mock
+except ImportError:
+    import mock
+import pytest
+
+from scrapy.crawler import Crawler
+from scrapy import Spider, signals
+from spidermon.contrib.scrapy.extensions import Spidermon
+
+
+@pytest.fixture
+def crawler():
+    def _crawler(extended_settings={}):
+        settings = {
+            'SPIDERMON_ENABLED': True,
+            'EXTENSIONS': {
+                'spidermon.contrib.scrapy.extensions.Spidermon': 500,
+            }
+        }
+        settings.update(extended_settings)
+        crawler = Crawler(Spider, settings=settings)
+        crawler.spider = Spider('dummy')
+        return crawler
+
+    return _crawler
+
+
+@pytest.fixture
+def suites():
+    return ['tests.fixtures.suites.Suite01']
+
+
+def test_spider_opened_suites_should_run(crawler, suites):
```

Comment:
As `crawler` argument is a function, maybe a better name for the fixture would be `get_crawler`...